site/docs/types/7/0770-Lineage-Mapping.md

```diff
@@ -9,11 +9,11 @@ hide:
 
 # 0770 - Lineage Mapping
 
-Lineage Mapping identifies elements that are [equivalent to each other in the lineage graph](/egeria-docs/features/lineage-mangement/#lineage-mapping).  It is used to stitch the lineage graph together.
+Lineage Mapping identifies elements that are [equivalent to each other in the lineage graph](/egeria-docs/features/lineage-management/overview/#the-lineage-graph). It is used to stitch the lineage graph together.
```

Comment:
I think the sub section should be #stitching

Reply:
As we are discussing about Lineage mapping, where will this section, #stiching go?

Reply:
The stitching section is on the lineage management page ... so /egeria-docs/features/lineage-management/overview/#stitching